pyproject.toml

```
@@ -143,6 +143,8 @@ filterwarnings = [
     "ignore:non-integer arguments to randrange\\(\\) have been deprecated since Python 3\\.10 and will be removed in a subsequent version:DeprecationWarning",
     "ignore:Deprecated call to `pkg_resources.declare_namespace:DeprecationWarning",
     "ignore:The --rsyncdir command line argument and rsyncdirs config variable are deprecated.:DeprecationWarning",
+    "ignore:Converting a tensor to a NumPy array might cause the trace to be incorrect.",
```

I wonder if ignoring these warnings for the whole project is a good thing

---

It's for pytest only

---

ah yes indeed, but still these warnings seem pretty important no ?

---

Yes! IMO these warning should be runtime errors. Not sure why torch allows exporting to onnx with broken forward.

But in pytest, warnings == failures. So we have to filter these.

---

but if these warnings should be considered warnings, then maybe it's best if we add a `with pytest.raise` for them instead of ignoring them no ? 🤔

---

This isn't a warnings raised from CML so I see no issue in just ignoring it.
Using `with pytest.raise` would mean that we want the warning to be raised.

---

yes ok you're right, fine  for me then !